src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -36,7 +36,9 @@ function maColumn($state, $anchorScroll, $compile, Configuration, FieldViewConfi
             scope.datastore = scope.datastore();
             scope.field = scope.field();
             scope.entry = scope.entry();
-            scope.value = scope.entry.values[scope.field.name()];
+            scope.value = typeof scope.entry === 'undefined'
+                ? ''
+                : scope.entry.values[scope.field.name()];
```

Comment:
`scope.value = scope.entry.values ? scope.entry.values[scope.field.name()] : '';` is better IMO.